docs/_interface/mining.md

```diff
@@ -1,191 +1,204 @@
+and installation instructions are available from the [Ethminer Github](https://github.com/ethereum-mining/ethminer/#build). Standalone executables are available for Linux, macOS and Windows.
 
-To monitor the syncing, in another terminal you can `attach` the geth JavaScript console to the running node like so:
+### Using Ethminer with Geth
 
-    geth attach http://127.0.0.1:8545
+An account to receive block rewards must first be created or imported into Geth. To create a new account:
 
-and then at the > prompt type
+```shell
+geth account new
+```
 
-    eth.syncing
+It is extremely important that the password is backed up safely and securely. The funds cannot be retrieved if the password is lost or forgotten. More detailed instructions on creating an importing accounts are available on the [Account Management](/docs/interface/managing-your-accounts) page.
 
-You'll see something like the example output below -- it's a two stage process as described in much more detail in our [FAQ](../FAQ).
-In the first stage, the difference between the "currentBlock" and the "highestBlock" will decrease until they are almost equal. It will then look stuck and appear as never becoming equal.  But you should see "pulledStates" rising to equal "knownStates." When both are equal, you are synced.
 
-Example output of first stage of block downloading:
+With an account in the Geth keystore, Geth can be started. The account address can be provided to `--mining.etherbase`. This instructs Geth todirect any block rewards to this address. Once started, Geth will sync the blockchain. If Geth has not connected to this network before, or if the data directory has been deleted, this can take several days. Also, enable HTTP traffic with the `--http` command.
 
-    {
-      currentBlock: 10707814,
-      highestBlock: 13252182,
-      knownStates: 0,
-      pulledStates: 0,
-      startingBlock: 3809258
-}
+```shell
+geth --http --miner.etherbase 0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10
+```
 
-You will import up to the `highestBlock` and `knownStates`. Block importing will stop `~64` blocks behind head and finish importing states. 
+The progress of the blockchain syncing can be monitored by attaching a JavaScript console in another terminal. More detailed information about the console can be found on the [Javascript Console](/docs/interface/javascript-console) page. To attach and open a console:
 
-Once all `states` are downloaded, `geth` will switch into a full node and sync the remaining `~64` blocks fully, as well as new ones. In this context, `eth.syncing` returns false once synced.
+```shell
+geth attach http://127.0.0.1:8545
+```
 
-Now we're ready to start mining. In a new terminal session, run ethminer and connect it to geth:
+Then in the console, to check the sync progress:
 
-OpenCL
+```shell
+eth.syncing
+```
 
-    ethminer -G -P http://127.0.0.1:8545
+If the sync is progressing correctly the output will look similar to the following:
 
-CUDA
+```terminal
```

Comment:
Not anymore, we changed the format of this output: 
```
> eth.syncing
{
  currentBlock: 13891665,
  healedBytecodeBytes: 0,
  healedBytecodes: 0,
  healedTrienodeBytes: 0,
  healedTrienodes: 0,
  healingBytecode: 0,
  healingTrienodes: 0,
  highestBlock: 14640000,
  startingBlock: 13891665,
  syncedAccountBytes: 0,
  syncedAccounts: 0,
  syncedBytecodeBytes: 0,
  syncedBytecodes: 0,
  syncedStorage: 0,
  syncedStorageBytes: 0
}
```

Reply:
ok, thank you!